frontend/src/local_cache.rs

```diff
@@ -8,6 +8,7 @@ use wasm_bindgen::UnwrapThrowExt;
 struct EventStore {
     likes: HashSet<i64>,
     unscreened: Vec<QuestionItem>,
+    premium_banner_collapsed: bool,
```

Comment:
its crucial to mark this `#[serde(default)]` so it won't break backwards compatibility with existing data stored in users browsers that does not have this field present

Reply:
👍 addressed in https://github.com/liveask/liveask/pull/82/commits/8a726eeeeed4d6a659f86f7c947258e4088031fd